tests/kubeless-integration/Makefile

```diff
@@ -0,0 +1,33 @@
+APP_NAME = kubeless-integration-tests
+IMG = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME)
+TAG = $(DOCKER_TAG)
+
+
+resolve: 
+	dep ensure -vendor-only -v
+
+validate:
+	go build -o golint-vendored ./vendor/github.com/golang/lint/golint
+	./golint-vendored
+	rm golint-vendored
+
+build:
+	go generate ./...
+	CGO_ENABLED=0 go build -o ./bin/app $(buildpath)
+
+build-image:
+	docker build -t $(APP_NAME):latest .
+
+push-image:
+	docker tag $(APP_NAME) $(IMG):$(TAG)
+	docker push $(IMG):$(TAG)
```

Comment:
no particular reason, I am just following the pattern we have in example for migration. AFAIK this will be reviewed by prow team again and they will unify everything.

Reply:
fine for me, as long as the result will be somehow consistent